yarn-project/p2p/src/services/peer-manager/peer_manager.ts

```diff
@@ -140,7 +140,7 @@ export class PeerManager implements PeerManagerInterface {
 
     if (this.config.preferredPeers) {
       const preferredPeersEnrs: ENR[] = this.config.preferredPeers.map(enr => ENR.decodeTxt(enr));
-      await Promise.all(preferredPeersEnrs.map(enr => enr.peerId()))
+      await Promise.all(preferredPeersEnrs.map(enr => enr.peerId))
```

Comment:
I'm slightly confused. This would suggest that `peerId` is a promise? But further down this file it seems it's not a promise?

Or do we need to remove all the promise code here?

Reply:
Good catch! We need to remove the `Promise` code, will fix!

Reply:
@PhilWindle  done!

Reply:
Same change required for `privatePeers` and `trustedPeers`?

Reply:
@PhilWindle  done2! https://github.com/AztecProtocol/aztec-packages/pull/15911/commits/c25157c01f847ab33bd896d55e53f783a172aec4

P.S.
CI is still failing on this one